app/population/view/daily.njk

```diff
@@ -0,0 +1,114 @@
+{% extends "layouts/base.njk" %}
+{% from "panel/macro.njk" import appPanel %}
+{% from "data/macro.njk" import appData %}
+
+{%  macro statsEntry(property, value) %}
+  {%  set key = "population::daily_view.labels." + property %}
+  {%  set label = t(key) %}
+
+  <h4 class="govuk-heading-s">{{ label }}</h4>
+  <p class="govuk-body">{{ value }}</p>
+{% endmacro %}
+
+{% macro statsGroup(headingProperty, stats) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  {%  for stat in stats %}
+    {{  statsEntry(stat.property, stat.value) }}
+  {% endfor %}
+{% endmacro %}
+
+{% macro transfersGroup(headingProperty, transfers) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  <p class="govuk-body">
+    {{  transfers }}
+  </p>
+{% endmacro %}
+
+{% set displayDate = details.date if details.date else date | formatDateWithRelativeDay %}
+
+{% block pageTitle %}
+  {{ t("population::daily_view.page_title_with_date", {
+    date: displayDate
+  }) }}
+{% endblock %}
+
+
+{% block content %}
+  <header class="govuk-grid-row govuk-!-margin-bottom-7">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">
+        <span class="govuk-caption-xl">
+            {{ t("population::daily_view.subheading") }}
+            </span>
+        <span class="govuk-!-display-block">{{ displayDate }}</span>
+      </h1>
+    </div>
+  </header>
+
+  {%  call appPanel({classes: 'govuk-!-margin-bottom-4'}) %}
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
```

Comment:
It feels like it might be more appropriate for this to be `two-thirds` like above. Looking at the screenshots it feels like it wraps too soon in some cases when this is only `one-half`.

Reply:
Replaced heading grid size with `two-thirds`